plugin/trino-druid/src/test/resources/ingestion-index.tpl

```diff
@@ -0,0 +1,59 @@
+ {
+     "type": "index",
+     "spec": {
+         "dataSchema": {
+             "dataSource": "${datasource}",
+             "parser": {
+                 "type": "string",
+                 "parseSpec": {
+                     "format": "tsv",
+                     "timestampSpec": {
+                         "column": "${timestampSpec.column}",
+                         "format": "${timestampSpec.format}"
+                     },
+                     "columns": [
+                        "${timestampSpec.column}",
+                        <#list columns as column>
+                        "${column.name}"<#sep>, </#sep>
+                        </#list>
+                     ],
+                     "dimensionsSpec": {
+                         "dimensions": [
+                            <#list columns as column>
+                            {
+                                "name": "${column.name}",
+                                "type": "${column.type}"
+                            }<#sep>,
+                            </#list>
+                         ]
+                     }
+                 }
+             },
+             "granularitySpec": {
+                 "type": "uniform",
+                 "intervals": [
+                     "1992-01-02/2028-12-01"
```

Comment:
Why this specific range? Can we have an even larger range? For example we might want to test dates in the far future and dates before the Gregorian-Julian switch (as we do for other connectors).

Reply:
I copied this from other index task files, they are `1992-01-02/1998-12-01`.
I may try `1001-01-01/9999-12-31`

Reply:
I assume we'll get a meaningful error message if someone in future adds a test for a date outside of the current range? If yes, then let's keep it as it is.